packages/@rnw-scripts/doxysaurus/src/markdown.ts

```diff
@@ -0,0 +1,549 @@
+        } else {
+          return log(
+            `Warning: Unknown parameterlist kind {${element.$.kind}}.`,
+          );
+        }
+      case 'parameteritem':
+        return write('\n* ', element.$$);
+      case 'parametername':
+        return writeCode('`', element.$$, '` ');
+      case 'computeroutput':
+        return writeCode('`', element.$$, '`');
+      case 'programlisting':
+        return writeCode('\n```cpp\n', element.$$, '```\n');
+      case 'codeline':
+        return write(element.$$, '\n');
+      case 'xrefsect':
+        return write('\n> ', element.$$);
+      case 'simplesect':
+        if (element.$.kind === 'attention') {
+          return write('> ', element.$$);
+        } else if (element.$.kind === 'return') {
+          return write(index ? '\n\n' : '', '### Returns\n\n', element.$$);
+        } else if (element.$.kind === 'see') {
+          return write('**See also**: ', element.$$);
+        } else {
+          log(`Warning: [element.$.kind=${element.$.kind}]: not supported.`);
+          return;
+        }
+      case 'formula':
+        let s = (element._ || '').trim();
+        if (s.startsWith('$') && s.endsWith('$')) {
+          return write(s);
+        }
+        if (s.startsWith('\\[') && s.endsWith('\\]')) {
+          s = s.substring(2, s.length - 2).trim();
+        }
+        return write('\n$$\n' + s + '\n$$\n');
+      case 'preformatted':
+        return writeCode('\n<pre>', element.$$, '</pre>\n');
+      case 'sect1':
+      case 'sect2':
+      case 'sect3':
+      case 'sect4':
+        return writeWithContext(element, index ? '\n\n' : '', element.$$);
+      case 'title':
+        const level = Number((last(context)?.['#name'] || '0').slice(-1));
+        return write('#'.repeat(level), ' ', element._);
+      case 'heading':
+        sb.removeLastIf(' ');
+        return write('#'.repeat(Number(element.$.level || 0)), ' ', element.$$);
+      case 'hruler':
+        return write('---');
+      case 'ref':
+        return refLink(element);
+      case 'ulink':
+        return link(toMarkdown(element.$$), element.$.url);
+      case 'xreftitle':
+        return write('**', element.$$, ':** ');
+      case 'row':
+        write('\n', escapeRow(toMarkdown(element.$$, linkResolver)));
+        if ((element.$$[0] as DoxDescriptionElement).$.thead === 'yes') {
+          element.$$.forEach((_, i) => {
+            write(i ? ' | ' : '\n', '---------');
+          });
+        }
+        break;
+      case 'entry':
+        return write(escapeCell(toMarkdown(element.$$, linkResolver)), '|');
+      case 'anchor':
+      case 'highlight':
+      case 'table':
+      case 'parameterdescription':
+      case 'parameternamelist':
+      case 'xrefdescription':
+      case 'verbatim':
+      case undefined:
+        return write(element.$$);
+
+      default:
+        log(
+          `Warning: [element[['#name'=${element['#name']}]]]: not supported.`,
```

Comment:
nit: I'd prefer error and have a list of explicit cases we know about that warn so we conciously ignore them.
Again I'm a typoholic :D

Reply:
The warning here is intentional. I do not want developers to be prevented from generating docs just because we see an unknown tag.